tools/report-converter/codechecker_report_converter/report/reports.py

```diff
@@ -17,8 +17,8 @@
 
 
 class GenericSuppressHandler:
-    get_suppressed: Callable[[Any, Report], bool]
-    store_suppress_bug_id: Callable[[Any, str, str, str, str], bool]
+    get_suppressed: Callable[[Report], bool]
```

Comment:
Please do this in a separate pr.

Reply:
This change is related to mypy upgrade, so I would leave it here.